packages/bson-bench/package.json

```diff
@@ -16,11 +16,11 @@
   "keywords": [],
   "author": "The MongoDB NodeJS Team <dbx-node@mongodb.com>",
   "license": "Apache-2.0",
-  "dependencies": {
-    "bson": "4.7"
-  },
   "directories": {
     "lib": "lib",
     "test": "test"
+  },
+  "devDependencies": {
+    "bson": "4.7"
```

Comment:
Moved to dev deps so that we have it available to test installing a local bson version